config/core/configmaps/default-pingsource.yaml

```diff
@@ -23,5 +23,5 @@ metadata:
     knative.dev/example-checksum: "6eaeecba"
```

Comment:
I already commented on that in the previous PR but it wasn't addressed. There is no example block, so this needs to be removed.